slave/components/assembler/assembler.go

```diff
@@ -89,7 +89,24 @@ func (a *asmbler) Process() error {
 		Cmd:     constants.AvailableAssemblers[a.assemblerName].Comm(a.filePath, a.outPath),
 		Volumes: map[string]struct{}{},
 	}
-	resp, err := a.dClient.ContainerCreate(a.ctx, ctConfig, nil, nil, "")
+	hostConfig := &container.HostConfig{
+		Mounts: []mount.Mount{
+			{ // Binding the file provided by the user to the docker container
+				Type:		mount.TypeBind,
+				Source:		a.filePath,
+				Target:		"/raw_sequence_input.fastq",
+				ReadOnly:	false,
+			},
+			{ // Binding the output directory provided by the user to the docker container
+				Type:		mount.TypeBind,
+				Source:		a.outPath,
+				Target:		"/output",
```

Comment:
This could be added to `constants.go`, close to `MegaHit`; maybe group them together and leave a general comment to make it look like a section?